personal-brand/personal-branding.md

```diff
@@ -69,3 +68,8 @@ Tweet out about your new blog on Twitter using the [#codenewbie](https://twitter
 ### Check for Understanding
 
 Send your first blog post to a peer and ask for comments and 2 ways you can improve your post or your site.
+
+### Supplemental Resources
+
+- [templates.netlify.com](http://templates.netlify.com/) - It consist of variety of templates which can be used for your website
```

Comment:
Lists a variety of website templates.

Reply:
@alodahl I have not understood what change am I supposed to do in this sentence.

Reply:
change :  "It consist of variety of templates which can be used for your website" -->

"Lists a variety of website templates."